<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крестики-нолики</string>
  </property>
  <widget class="QRadioButton" name="X_RadioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="O_RadioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="new_game_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
